cranelift/codegen/src/ir/types.rs

```diff
@@ -157,7 +152,7 @@ impl Type {
     /// Scalar types are all converted to `b1` which is usually what you want.
     pub fn as_bool(self) -> Self {
```

Comment:
Perhaps we could remove the `.as_bool()` helper altogether?

Reply:
How would you feel about renaming it to `.as_truthy()` instead? It does convert types like `F32` to `I32`, so it would be convenient to keep something like it around.